ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -57,6 +57,11 @@ public static class ValidationMessages
         public const string ValidTreatmentOptions = "Short course and MDR treatment cannot both be true";
         public const string MDRCantChange = "You cannot change the value of this field because an MDR Enhanced Surveillance Questionnaire exists. Please contact NTBS@phe.gov.uk";
         public const string MDRDuration = "Please enter a whole number of months, or a range, for example, 12, 12-18 or 24+";
+        public const string IsPostMortemButStartedTreatment =
+            "Unable to save value of ‘Diagnosis after death’ field due to conflicting information on the record: Started treatment is set to true";
```

Comment:
Maybe "The patient has started treatment" rather than "Started treatment is set to true"